chapter_linear-networks/linear-regression_vn.md

```diff
@@ -577,7 +577,7 @@ Vectorizing code often yields order-of-magnitude speedups.
 Moreover, we push more of the math to the library and need not write as many calculations ourselves, reducing the potential for errors.
 -->
 
-Bạn có thể nhận thấy rằng, phương pháp thứ hai nhanh hơn rất nhiều lần so với phương pháp đầu tiên.
+Bạn có thể nhận thấy rằng, phương pháp thứ hai nhanh hơn rất nhiều lần so với phương pháp thứ nhất.
 Việc vector hóa thường tăng tốc độ tính toán lên nhiều lần.
```

Comment:
tăng nhiều lần nghe chưa đã lắm. Em cũng không chắc `order of magnitude` dịch là gì trong tiếng việt.
```suggestion
Việc vector hóa thường tăng tốc độ tính toán lên nhiều bậc.
```

Reply:
một `order of magnitude` là 10x nhé, dịch nôm na là `tăng gấp hàng chục hàng trăm lần`.
Mình thấy để thế này cũng được, nhưng nếu muốn sát nghĩa hay nghe cho hay hơn thì thế này:
```suggestion
Việc vector hóa thường tăng tốc độ tính toán lên hàng chục, hàng trăm lần.
```